libs/SalesforceSDKCore/SalesforceSDKCore/Classes/OAuth/SFOAuthCoordinator.m

```diff
@@ -568,9 +569,11 @@ - (void)beginUserAgentFlow {
                 json = [NSJSONSerialization JSONObjectWithData:data options:0 error:&jsonError];
                 if (nil == jsonError && [json isKindOfClass:[NSDictionary class]]) {
                     NSDictionary *dict = (NSDictionary *)json;
-                    if (dict[kSFOAuthAccessToken]) {
+                    [self updateCredentials:dict];
+                    if (self.credentials.accessToken && self.credentials.apiUrl) {
+                        NSString *baseUrlString = [self.credentials.apiUrl absoluteString];
```

Comment:
`apiUrl` will ensure Community URL is used if available, instance URL otherwise.